docs/Tutorials - start here/yolov5.md

```diff
@@ -0,0 +1,52 @@
+# How to Use Innofw Framework for Object Detection using YoloV5
+
+In this tutorial, we will show you how to use Innofw framework for object detection using YoloV5. We will show you how to create a configuration file for your dataset, how to create a configuration file for your training, how to train your model, how to evaluate your model and how to make an inference on your model.
+
+There are different variations of yolov5 library. For example, you can use yolov5s, yolov5m, yolov5l or yolov5x. The letter at the end of the name indicates the size of the model. The bigger the letter, the bigger the model. The bigger the model, the more accurate it is. However, the bigger the model, the slower it is. In this tutorial, we will use yolov5s. You can change the model by changing the `arch` parameter in the file `innofw\config\models\detection\yolov5.yaml`.
```

Comment:
I think this will be more appropriate `There are different variations of the yolov5 model.`

Reply:
The statement, "The bigger the letter, the bigger the model. The bigger the model, the more accurate it is," seems to oversimplify the matter. It would be more precise to say that the models for YOLOv5 follow a certain order: 's', 'm', 'l', 'x'. This sequence signifies a progression from the smallest and fastest model to the largest and slowest one. It's important to note that while larger models tend to offer higher accuracy, they also demand more computational resources and take longer to process.

Reply:
I think it is better to replace `innofw\config\models\detection\yolov5.yaml` with `config\models\detection\yolov5.yaml`